docs/ai/README.md

```diff
@@ -0,0 +1,25 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: AI
```

Comment:
nit: since it's LLM semconv, I think it should be in the llm folder and has LLM title